docs/developer/overview-developer.md

```diff
@@ -1,11 +1,23 @@
 ---
 title: Developer Documentation Overview
+summary: This article is an overview of the developer documentation for the Run:ai platform.
+authors:
+    - Jason Novich
+date: 2024-Apr-10
 ---
 
-# Overview: Developer Documentation
+Developers can access Run:ai through various programmatic interfaces.
 
-Developers can access Run:ai through various programmatic interfaces. 
+## API Support
 
+The endpoints and parameters specified in the API reference are the ones that are officially supported by Run:ai. Endpoints and parameters that are **NOT** listed in the reference are not officially supported.
+
+The endpoints and parameters specified in the API reference are the ones that are officially supported by Run:ai. Endpoints and parameters that are **NOT** listed in the reference are not officially supported.
```

Comment:
@jasonnovichRunAI it seems like this sentence is duplicated, and the sentence that was here was deleted...